scriptHelper.js

```diff
@@ -14,30 +16,159 @@ function addDestinationInfo(document, name, diameter, star, distance, moons, ima
+        //     </ol>
+        // `;
+        pilotStatus.innerHTML = `Pilot ${pilot} is ready for launch`;
+        copilotStatus.innerHTML = `Co-pilot ${copilot} is ready for launch`;
+        fuelLevelStatus.innerHTML = `Fuel level high enough for launch`;
+        cargoLevelStatus.innerHTML = `Cargo mass too heavy for launch`;
+        h2.innerHTML = `Shuttle Not Ready for Launch`;
+        h2.style.color = 'rgb(199, 37, 78)';
+    } else if (cargoLevel > 10000 && fuelLevel < 10000){
+                console.log("Cargo Mass too high and Fuel Level low");
+                list.style.visibility = "visible";
+                // list.innerHTML = `
+                //     <ol>
+                //         <li id="pilotStatus" data-testid="pilotStatus">Pilot ${pilot} is ready for launch</li>
+                //         <li id="copilotStatus" data-testid="copilotStatus">Co-pilot ${copilot} is ready for launch</li>
+                //         <li id="fuelStatus" data-testid="fuelStatus">Fuel level too low for launch</li>
+                //         <li id="cargoStatus" data-testid="cargoStatus">Cargo mass too heavy for launch</li>
+                //     </ol>
+                // `;
+                pilotStatus.innerHTML = `Pilot ${pilot} is ready for launch`;
+                copilotStatus.innerHTML = `Co-pilot ${copilot} is ready for launch`;
+                fuelLevelStatus.innerHTML = `Fuel level too low for launch`;
+                cargoLevelStatus.innerHTML = `Cargo mass too heavy for launch`;
+                h2.innerHTML = `Shuttle Not Ready for Launch`;
+                h2.style.color = 'rgb(199, 37, 78)';
+    } else if (fuelLevel >= 10000 && cargoLevel <= 10000 && validate(pilot, copilot, fuelLevel, cargoLevel) === "Validated"){
+        console.log("Launch Checklist when everything is good to go.");
+                console.log(list);
+                list.style.visibility = "visible";
+                // list.innerHTML = `
+                //     <ol>
+                //         <li id="pilotStatus" data-testid="pilotStatus">Pilot ${pilot} is ready for launch</li>
+                //         <li id="copilotStatus" data-testid="copilotStatus">Co-pilot ${copilot} is ready for launch</li>
+                //         <li id="fuelStatus" data-testid="fuelStatus">Fuel level high enough for launch</li>
+                //         <li id="cargoStatus" data-testid="cargoStatus">Cargo mass low enough for launch</li>
+                //     </ol>
+                // `;
+                pilotStatus.innerHTML = `Pilot ${pilot} is ready for launch`;
+                copilotStatus.innerHTML = `Co-pilot ${copilot} is ready for launch`;
+                fuelLevelStatus.innerHTML = `Fuel level high enough for launch`;
+                cargoLevelStatus.innerHTML = `Cargo mass low enough for launch`;
+                h2.innerHTML = `Shuttle is Ready for Launch`;    
+                h2.style.color = 'rgb(65, 159, 106)';
+    }
```

Comment:
```suggestion
  let isValidated = validate(pilot, copilot, fuelLevel, cargoLevel);

  if (isValidated) {
    if (fuelLevel < 10000 && cargoLevel < 10000){
        console.log("Fuel Level low");
        list.style.visibility = "visible";
        // list.innerHTML = `
        //     <ol>
        //         <li id="pilotStatus" data-testid="pilotStatus">Pilot ${pilot} is ready for launch</li>
        //         <li id="copilotStatus" data-testid="copilotStatus">Co-pilot ${copilot} is ready for launch</li>
        //         <li id="fuelStatus" data-testid="fuelStatus">Fuel level too low for launch</li>
        //         <li id="cargoStatus" data-testid="cargoStatus">Cargo mass low enough for launch</li>
        //     </ol>
        // `;
        console.log(pilotStatus.innerHTML);
        pilotStatus.innerHTML = `Pilot ${pilot} is ready for launch`;
        console.log(pilotStatus.innerHTML);
        copilotStatus.innerHTML = `Co-pilot ${copilot} is ready for launch`;
        fuelLevelStatus.innerHTML = `Fuel level too low for launch`;
        cargoLevelStatus.innerHTML = `Cargo mass low enough for launch`;
        h2.innerHTML = `Shuttle Not Ready for Launch`;
        console.log(h2.innerHTML);
        h2.style.color = 'rgb(199, 37, 78)';


    } else if (cargoLevel > 10000 && fuelLevel >= 10000){
        console.log("Cargo Mass too high");
        list.style.visibility = "visible";
        // list.innerHTML = `
        //     <ol>
        //         <li id="pilotStatus" data-testid="pilotStatus">Pilot ${pilot} is ready for launch</li>
        //         <li id="copilotStatus" data-testid="copilotStatus">Co-pilot ${copilot} is ready for launch</li>
        //         <li id="fuelStatus" data-testid="fuelStatus">${fuelLevel} liters</li>
        //         <li id="cargoStatus" data-testid="cargoStatus">Cargo mass too heavy for launch</li>
        //     </ol>
        // `;
        pilotStatus.innerHTML = `Pilot ${pilot} is ready for launch`;
        copilotStatus.innerHTML = `Co-pilot ${copilot} is ready for launch`;
        fuelLevelStatus.innerHTML = `Fuel level high enough for launch`;
        cargoLevelStatus.innerHTML = `Cargo mass too heavy for launch`;
        h2.innerHTML = `Shuttle Not Ready for Launch`;
        h2.style.color = 'rgb(199, 37, 78)';
    } else if (cargoLevel > 10000 && fuelLevel < 10000){
                console.log("Cargo Mass too high and Fuel Level low");
                list.style.visibility = "visible";
                // list.innerHTML = `
                //     <ol>
                //         <li id="pilotStatus" data-testid="pilotStatus">Pilot ${pilot} is ready for launch</li>
                //         <li id="copilotStatus" data-testid="copilotStatus">Co-pilot ${copilot} is ready for launch</li>
                //         <li id="fuelStatus" data-testid="fuelStatus">Fuel level too low for launch</li>
                //         <li id="cargoStatus" data-testid="cargoStatus">Cargo mass too heavy for launch</li>
                //     </ol>
                // `;
                pilotStatus.innerHTML = `Pilot ${pilot} is ready for launch`;
                copilotStatus.innerHTML = `Co-pilot ${copilot} is ready for launch`;
                fuelLevelStatus.innerHTML = `Fuel level too low for launch`;
                cargoLevelStatus.innerHTML = `Cargo mass too heavy for launch`;
                h2.innerHTML = `Shuttle Not Ready for Launch`;
                h2.style.color = 'rgb(199, 37, 78)';
    } else if (fuelLevel >= 10000 && cargoLevel <= 10000 && validate(pilot, copilot, fuelLevel, cargoLevel) === "Validated"){
        console.log("Launch Checklist when everything is good to go.");
                console.log(list);
                list.style.visibility = "visible";
                // list.innerHTML = `
                //     <ol>
                //         <li id="pilotStatus" data-testid="pilotStatus">Pilot ${pilot} is ready for launch</li>
                //         <li id="copilotStatus" data-testid="copilotStatus">Co-pilot ${copilot} is ready for launch</li>
                //         <li id="fuelStatus" data-testid="fuelStatus">Fuel level high enough for launch</li>
                //         <li id="cargoStatus" data-testid="cargoStatus">Cargo mass low enough for launch</li>
                //     </ol>
                // `;
                pilotStatus.innerHTML = `Pilot ${pilot} is ready for launch`;
                copilotStatus.innerHTML = `Co-pilot ${copilot} is ready for launch`;
                fuelLevelStatus.innerHTML = `Fuel level high enough for launch`;
                cargoLevelStatus.innerHTML = `Cargo mass low enough for launch`;
                h2.innerHTML = `Shuttle is Ready for Launch`;    
                h2.style.color = 'rgb(65, 159, 106)';
    }
  }
```
I love that you're using a separate function to separate the validation piece for the inputs! I'd recommend moving your `validate()` function call inside `formSubmission()` to prevent it from getting called twice.